src/ucp/wireup/wireup.c

```diff
@@ -1159,7 +1128,8 @@ ucs_status_t ucp_wireup_init_lanes(ucp_ep_h ep, unsigned ep_init_flags,
         return status;
     }
 
-    ucp_wireup_check_config_intersect(ep, &key, &replay_pending_queue);
+    ucp_wireup_check_config_intersect(ep, &key, reuse_lane_map,
+                                      &replay_pending_queue);
```

Comment:
can we just return a **bit**map of which lanes need to connect (e.g which lanes are new)?
calculate the map inside ucp_wireup_check_config_intersect as part of lane reuse logic

Reply:
done